service/src/main/java/bio/terra/workspace/db/exception/CloudContextNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.workspace.db.exception;
+
+import bio.terra.common.exception.NotFoundException;
+
+public class CloudContextNotFoundException extends NotFoundException {
```

Comment:
Doesn't seem like this is used in this PR?

Reply:
I will remove it. Thanks